src/data_race.rs

```diff
@@ -824,8 +824,7 @@ impl VClockAlloc {
         // Throw the data-race detection.
         throw_ub_format!(
             "Data race detected between {} on {} and {} on {}, memory({:?},offset={},size={})\
-            \n\t\t -current vector clock = {:?}\
-            \n\t\t -conflicting timestamp = {:?}",
+            \n(current vector clock = {:?}, conflicting timestamp = {:?})",
```

Comment:
@JCTyblaidd I had to change your messages here since rustc now ICEs when there is a tab in the message:
```
thread 'rustc' panicked at 'assertion failed: self.text.iter().all(|r| !r.contains(&\'\\t\'))', compiler/rustc_errors/src/styled_buffer.rs:18:9
```